typescript/karma.conf.js

```diff
@@ -29,6 +29,7 @@ module.exports = function (config) {
       subdir: '.',
       reporters: [
         { type: 'html' },
+        { type: 'cobertura' },
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/newsbridge/sandbox/pullRequest?prid=7539937)
```suggestion
        { type: "cobertura" },
```